aggregator/src/tests/compression.rs

```diff
@@ -0,0 +1,99 @@
+use std::{fs, path::Path, process};
+
+use ark_std::{end_timer, start_timer, test_rng};
+use halo2_proofs::{
+    dev::MockProver,
+    halo2curves::bn256::{Bn256, Fr},
+    poly::commitment::Params,
+};
+use snark_verifier::{
+    loader::halo2::halo2_ecc::halo2_base::{halo2_proofs, utils::fs::gen_srs},
+    pcs::kzg::{Bdfg21, Kzg},
+};
+use snark_verifier_sdk::{
+    evm_verify, gen_evm_proof_shplonk, gen_evm_verifier, gen_pk, gen_snark_shplonk,
+    verify_snark_shplonk, CircuitExt,
+};
+
+use crate::{
+    compression_layer_evm, compression_layer_snark, layer_0, tests::mock_chunk::MockChunkCircuit,
+    CompressionCircuit,
+};
+
+#[test]
```

Comment:
seems this test increased CI time several minutes? what about ignore it by default? (i don't know which new test added most time)

Reply:
Yes this test takes quite long. It is ignored for now. 

Note that now aggregation crate __does not__ have any meaningful unit tests with CI.